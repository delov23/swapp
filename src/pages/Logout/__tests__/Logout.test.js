import React from 'react';
import { shallow, mount } from 'enzyme';
import { BrowserRouter as Router } from 'react-router-dom';

import { Logout } from '../';

describe('<Logout />', () => {
  it('should match snapshot', () => {
    expect(shallow(<Logout />)).toMatchSnapshot();
  });

  it('should redirect upon arrival', () => {
    localStorage.setItem('token', 'example');

    const wrapper = mount(
      <Router>
        <Logout />
      </Router>,
    );

    wrapper.update();

    expect(localStorage.getItem('token')).toBeFalsy();
  });
});
