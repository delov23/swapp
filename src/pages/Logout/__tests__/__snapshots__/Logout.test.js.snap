// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logout /> should match snapshot 1`] = `
<Redirect
  to="/login"
/>
`;
